builtin/logical/pki/path_sign_issuers.go

```diff
@@ -67,6 +67,24 @@ SHA-2-512. Defaults to 0 to automatically detect based on key length
 		},
 	}
 
+	fields["skid"] = &framework.FieldSchema{
+		Type:    framework.TypeString,
+		Default: 0,
```

Comment:
Should this not be a blank string instead of a 0?

Reply:
Ah, no, you're correct.